cluster/src/test/scala/io/snappydata/benchmark/snappy/tpch/SparkAppUsingJob.scala

```diff
@@ -63,6 +63,12 @@ object SparkAppUsingJob  extends SnappySQLJob {
     snc.dropTable("LINEITEM", ifExists = true)
     snc.dropTable("ORDERS", ifExists = true)
 
+    TPCHReplicatedTable.createPopulateRegionTable(usingOptionString, snc, tpchDataPath,
```

Comment:
Create tables in the same order for Snappy and Spark tests. Makes it easier to report results too.